int32 class_label
float64 x
float64 y
float64 thetha
